test/specs/app.spec.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import { expect } from 'chai';
```

Comment:
chai expect, should, assert could be added as test globals in your test/setup file
```global.chai = require('chai');
global.chai.use(require('chai-as-promised'));
global.should = chai.should();
global.expect = chai.expect;
global.assert = chai.assert;
```